lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -600,7 +600,9 @@ defmodule ExUnit.Assertions do
   end
 
   @doc """
-  Asserts that `value1` and `value2` differ by no more than `delta`.
+  Asserts that `value1` and `value2` differ by no more than `delta`. All `delta`s
```

Comment:
The paragraph of documentation should always be a short summary. The summary is used by tools like ExDoc.